packages/drivers/routerlicious-driver/src/documentService.ts

```diff
@@ -65,11 +72,19 @@ export class DocumentService implements api.IDocumentService {
             };
         }
 
+        const restWrapper = new BasicRestWrapper(
+            this.gitUrl,
+            {},
+            undefined,
+            {
+                Authorization: getAuthorizationTokenFromCredentials(credentials),
+            },
+        );
         const historian = new Historian(
             this.gitUrl,
             this.historianApi,
             this.disableCache,
-            credentials);
+            restWrapper);
```

Comment:
Breaking change detailed in [server/BREAKING.md](https://github.com/microsoft/FluidFramework/blob/main/server/BREAKING.md#historian-class-no-longer-handles-request-headers). This change maintains functionality (except `token` query param, which is no longer necessary and I will be making a PR to remove that in server/r11s/packages/services/srctenant.ts as well